src/components/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import styles from './Spinner.module.scss';
+import { PALMETTO_BRAND_COLOR_OPTIONS, PALMETTO_BRAND_COLOR_VALUES, PALMETTO_COLORS } from '../../lib/tokens';
+
+interface Props {
+  /**
+   * Custom classname to be applied to spinner container div.
+   */
+  className?: string;
+  /**
+   * Spinner color, pass down one of the options from the Palmetto 'brand' palette.
+   */
+  color?: PALMETTO_COLORS;
+  /**
+   * Size of the spinner.
+   */
+  size?: 'sm' | 'md' | 'lg';
+}
+
+const Spinner: React.FC<Props>= ({
+  color = 'dark',
+  className,
+  size = 'md',
+}) => {
+  const classes = classNames(className, styles.spinner);
+
+  const sizeInPixels = () => {
+    let pixels;
+    if (size === 'sm') pixels = '12';
+    if (size === 'md') pixels = '16';
+    if (size === 'lg') pixels = '24';
+
+    return pixels;
+  };
+
+  return (
+    <span className={classes}>
+      <svg
```

Comment:
It would be nice if we could store this svg in the images folder, and import using svgr. Could we set the width and height of the svg to 100% and wrap it in a parent div that has its dimensions constrained? Also, might it be possible to set the `color` on the container and have it cascade into the svg?